packages/blocks/src/page-block/edgeless/components/component-toolbar/component-toolbar.ts

```diff
@@ -383,7 +383,7 @@ export class EdgelessComponentToolbar extends WithDisposable(LitElement) {
     return html` <style>
         :host {
           position: absolute;
-          z-index: 1;
+          z-index: 3;
```

Comment:
There was a plan to make the z-index of widgets derived from CSS variables, we can do this in later refactoring.